cypress/e2e/helpers/surveyHelper.ts

```diff
@@ -1,21 +1,41 @@
 import "moment/min/locales";
 import { format } from "date-fns";
 
-import * as surveyStrings from "../strings/surveyStrings";
-import * as generalStrings from "../helpers/generalStrings";
+import {
+  CLOSE,
+  WHAT_IS_YOUR_WEIGHT_TODAY,
+  PLEASE_WRITE_FULL_NUMBER_DECIMAL,
+  REQUIRED,
+  LBS_00_0,
+  LBS_000_0,
+  LBS,
+  PLEASE_INSERT_VALUE_BETWEEN_60_700_LBS,
+  WHAT_IS_YOUR_BODY_FAT_PERCENTAGE_TODAY,
+  PLEASE_INSERT_VALUE_BETWEEN_2_60_PERCENT,
+  NEEAR001_BODY_COMPOSITION,
+  PERCENT,
+  PERCENT_NUMBER,
+  SERISD001_BODY_WEIGHT,
+  SURVEY_COMPLETED,
+} from "../strings/surveyStrings";
+
+import { NEXT, PREVIOUS, SUBMIT_SURVEY } from "../helpers/generalStrings";
 
 export const SERISD001_BODY_WEIGHT_ID: string =
   '[data-testid="SURVEY_CARD_SER-ISD-001 - Body Weight"]';
+export const NEEAR001_BODY_COMPOSITION_ID: string =
+  '[data-testid="SURVEY_CARD_NEAAR-001 - Body Composition"]';
 export const DATE_TEXT: string = "p.MuiTypography-body1.css-xhbppr";
 export const BUTTON: string = "button";
 export const VALUE_LBS_INPUT: string = "input.MuiInputBase-input";
-export const LBS_TEXT: string = "p.MuiTypography-root.css-10o8u6h";
-export const PROGRESS_BAR_50_IMG: string =
-  'span[role="progressbar"][aria-valuenow="50"]';
-export const PROGRESS_BAR_100_IMG: string =
-  'span[role="progressbar"][aria-valuenow="100"]';
+export const VALUE_TYPE: string = "p.MuiTypography-root.css-10o8u6h";
+export const PROGRESS_BAR_VALUE_ID = (progressValue: string): string => {
+  return `span[role="progressbar"][aria-valuenow="${progressValue}"]`;
+};
```

Comment:
When we have only one statement inside {} we don't need to add them
```suggestion
export const PROGRESS_BAR_VALUE_ID = (progressValue: string): string => `span[role="progressbar"][aria-valuenow="${progressValue}"]`;
```

Reply:
I didn't know, good tip, thanks!